--- setup.py.orig	2017-07-18 22:43:57.632873640 +0800
+++ setup.py	2017-07-18 22:44:16.355874726 +0800
@@ -27,7 +27,7 @@
     long_description=long_description(),
     url='https://github.com/RicterZ/BGmi',
     download_url='https://github.com/RicterZ/BGmi/tarball/master',
-    packages=find_packages(),
+    packages=find_packages(exclude=['tests']),
     package_data={'': ['LICENSES']},
     include_package_data=True,
     zip_safe=False,
